pkg/antctl/raw/multicluster/common/common.go

```diff
@@ -303,6 +304,100 @@ func CreateMemberToken(cmd *cobra.Command, k8sClient client.Client, name string,
 	return nil
 }
 
+func GetMemberToken(cmd *cobra.Command, k8sClient client.Client, name string, namespace string, file *os.File) error {
+	secret := &corev1.Secret{}
+	if err := k8sClient.Get(context.TODO(), types.NamespacedName{Name: name, Namespace: namespace}, secret); err != nil {
+		return err
+	}
+
+	if secret.Annotations[CreateByAntctlAnnotation] == "true" {
+		s := &corev1.Secret{
+			TypeMeta: metav1.TypeMeta{
+				APIVersion: "v1",
+				Kind:       "Secret",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name: name,
+			},
+			Data: secret.Data,
+			Type: corev1.SecretTypeOpaque,
+		}
+		if file != nil {
+			b, err := yaml.Marshal(s)
+			if err != nil {
+				return err
+			}
+			if _, err := file.Write([]byte("---\n")); err != nil {
+				return err
+			}
+			if _, err := file.Write(b); err != nil {
+				return err
+			}
+			fmt.Fprintf(cmd.OutOrStdout(), "Member token saved to %s\n", file.Name())
+		} else {
+			output.YamlOutput(s, cmd.OutOrStdout())
+		}
+	}
+
```

Comment:
I think you need a warning message like `Token *** created by antctl is not found` here